### PR TITLE
doc: add minutes for meeting 7 may 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

### CODE REVIEW

File: meetings/2020-05-07.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+* http: align with stream.Writable [#31818](https://github.com/nodejs/node/pull/31818)
+  * Vote in issue, achieved simple majority based on votes and abstains.
+    * Yes: 9 (Trott, jasnell, mcollina, BridgeAR, mhdawson, mmarchini, addaleax, ChALkeR, targos)
+    * No: 0
+    * Abstain: 4 (gabrielschulhof, BethGriggs, sam-github, tniessen)
+
+### nodejs/TSC
+
+* Nominations for TSC Chair - Open April 16 through 30th 2020 [#851](https://github.com/nodejs/TSC/issues/851)
+  * only one volunteera (Michael)
```
Comment (cjihrig) [inline meetings/2020-05-07.md:56]:
```suggestion
  * only one volunteer (Michael)

```

File: meetings/2020-05-07.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  * 10 +1s in issue for Michael to continue as Chair. Unless objection by Friday will proceed on that
+    basis.
+
+* Node.js future directions - any interest in online or in person summit? [#797](https://github.com/nodejs/TSC/issues/797)
+  * Michael will submit session for OpenJS World summit.[a]
```
Comment (cjihrig) [inline meetings/2020-05-07.md:61]: What is the `[a]`

File: meetings/2020-05-07.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+### nodejs/TSC
+
+* Nominations for TSC Chair - Open April 16 through 30th 2020 [#851](https://github.com/nodejs/TSC/issues/851)
+  * only one volunteer (Michael)
+
+  * 10 +1s in issue for Michael to continue as Chair. Unless objection by Friday will proceed on that
+    basis.
+
+* Node.js future directions - any interest in online or in person summit? [#797](https://github.com/nodejs/TSC/issues/797)
+  * Michael will submit session for OpenJS World summit.[a]
```
Comment (mhdawson) [inline meetings/2020-05-07.md:62]:
```suggest
  * Michael will submit session for OpenJS World summit.
```

File: meetings/2020-05-07.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2020-05-07.md]: Not sure , removed.

**Pull request merged**